api/pkg/template/license_test.go

```diff
@@ -590,11 +598,121 @@ func TestEngine_ChannelName_ChannelNotFound(t *testing.T) {
 		},
 	}
 
-	engine := NewEngine(config, WithLicense(license), WithReleaseData(releaseData))
+	engine := NewEngine(config, WithLicense(wrapLicense(license)), WithReleaseData(releaseData))
 
 	err := engine.Parse("{{repl ChannelName }}")
 	require.NoError(t, err)
 	_, err = engine.Execute(nil, WithProxySpec(&ecv1beta1.ProxySpec{}))
 	assert.Error(t, err)
 	assert.Contains(t, err.Error(), "channel unknown-channel-id not found in license")
 }
+
+func TestEngine_LicenseWrapper(t *testing.T) {
```

Comment:
what does this test?

Reply:
Right now I think only that it works with a `v1beta2` license, but I can just change one of the other test cases to handle that we we cover everything this test does.